playbooks/robusta_playbooks/node_disk_analysis.py

```diff
@@ -6,8 +6,13 @@
 import humanize
 
 
+class DiskAnalyzerParams(ActionParams):
+    show_pod_level_details: bool = True
```

Comment:
I would shorten it to: 
pod_level, container_level
or
pod_level_details, container_level_details

Reply:
@arikalon1 is `show_pods` and `show_containers` OK? I prefer having a verb (`show`) in the setting name.

Reply:
k